### PR TITLE
Update User Guide and Developer Guide

### PR DESCRIPTION
**Pull request by qiu-siqi**
^

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -361,8 +464,23 @@ Format: `theme THEME_NAME`
...
+
+Bibliotek shows `Application theme changed to: white` to indicate that your theme change is successful.
+
+The same goes for `theme light`:
+
+image::ThemeLight.png[width="700"]
+
+And `theme dark`:
+
+image::ThemeDark.png[width="700"]
```
Comment (takuyakanbr) [inline docs/UserGuide.adoc]: Temporarily modify `DarkTheme.css` on line 402 to `.pills .label {`

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -206,7 +207,7 @@ The `Model`,
 * stores a `UserPref` object that represents the user's preferences.
 * stores the Book Shelf data.
 * exposes an unmodifiable `ObservableList<Book>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
-* does not depend on any of the other four components.
+* does not depend on any of the other components.
 
 [[Design-Storage]]
 === Storage component
```
Comment (takuyakanbr) [inline docs/DeveloperGuide.adoc]: For figure 10, `XMLSerializableBookShelf`, put `Bo` on the next line

File: docs/UserGuide.adoc (hunk 1 above)
Comment (qiu-siqi) [reply docs/UserGuide.adoc]: thats exactly what it says?

Comment (takuyakanbr) [reply docs/UserGuide.adoc]: Oops. *line 410 to `.pills-small .label {`

**Pull request merged**